Changed "Our Department" description

JeroenDeDauw opened the pull request:
Main reason: all other descriptions where 2 lines and this one was 4,
which made the layout look quite silly.

Secondary reason: the other description was kinda fluff, and more about
how we work (and what we do (already under "Our Projects")) than who
we are.

JeroenDeDauw commented:
@samu-wmde does this look good to you?

JeroenDeDauw commented:
Ping @samu-wmde

tobijat commented:
I would change it to "We are currently consisting of 3 agile development teams." but @JeroenDeDauw 's suggestion is quite good already.

JeroenDeDauw commented:
Right. So you want to change

> We are 3 agile teams working at the Wikimedia DE office in Berlin.

into (I fixed the spelling/grammar)

> We currently consist out of 3 agile development teams.

I see two differences:

* Added emphasis on the present. Not sure why that is needed
* Removal of the location we are at. What motivated this removal?

JeroenDeDauw commented:
Since there has been no objection so far I will merge this and have created an issue to keep track of Sandra still needing to provide input: https://github.com/wmde/software.wikimedia.de/issues/19
